<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RBInfectionEditor</class>
 <widget class="QDialog" name="RBInfectionEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="lblName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Наименование</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="edtCode"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblCode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Код</string>
     </property>
     <property name="buddy">
      <cstring>edtCode</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="edtName"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="edtRegionalCode"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblRegionalCode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Региональный код</string>
     </property>
     <property name="buddy">
      <cstring>edtRegionalCode</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblMinimumTerm">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Минимальный срок</string>
     </property>
     <property name="buddy">
      <cstring>cmbMinimumTermUnit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cmbMinimumTermUnit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Д</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Н</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>М</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Г</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="edtMinimumTermCount">
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="pnlView" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="CInDocTableView" name="tblInfectionVaccines"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pnlInfectionMiminumTerms" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="CInDocTableView" name="tblInfectionMinimumTerms"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QCheckBox" name="chkAvailable">
     <property name="text">
      <string>В наличии</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtRegionalCode</tabstop>
  <tabstop>cmbMinimumTermUnit</tabstop>
  <tabstop>edtMinimumTermCount</tabstop>
  <tabstop>tblInfectionVaccines</tabstop>
  <tabstop>tblInfectionMinimumTerms</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RBInfectionEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RBInfectionEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
